keyboards/retne/bonsai/readme.md

```diff
@@ -0,0 +1,15 @@
+# Bonsai
+
+![Bonsai](https://i.imgur.com/XypHguN.jpg)
```

Comment:
```suggestion
![Bonsai](https://i.imgur.com/XypHguNh.jpg)
```

Reply:
Was this change an intended suggestion?  The new suggested url does not point to an image of the keyboard.

Reply:
It does, for me.  The "h" at the end changes which quality that imgur uses for the image.

Reply:
Ah interesting.  I looked at this on my phone before I sat down at the PC and the url didn't go anywhere on my phone.  I'll commit it.